docs/api.md

```diff
@@ -1151,8 +1151,13 @@ for details of this object
 
 <a name="parserConfiguration"></a>.parserConfiguration(obj)
 ------------
-`parserConfiguration()` allows you to configure yargs. See [yargs-parser's configuration](https://github.com/yargs/yargs-parser#configuration) for
-settings specific to `yargs-parser`.
+`parserConfiguration()` allows you to configure yargs-parser. 
+
+An object of options should be passed in.
```

Comment:
let's change this to:

`obj` accepts the following configuration options:

Reply:
done.